README.md

```diff
@@ -77,6 +77,7 @@ Is Ansistrano ready to be used? Here are some companies currently using it:
 * [Jolicode](http://jolicode.com/)
 * [Kidfund](http://link.kidfund.us/github "Kidfund")
 * [MEDIA.figaro](http://media.figaro.fr)
+* [Lumao SAS](https://lumao.eu)
```

Comment:
Sorry to be picky... it should go 1 line before

Reply:
Oh sorry, I corrected it